README.rst

```diff
@@ -121,9 +121,11 @@ Usage
     []
     >>> from foo.models import BarTime
     >>> BarTime(name="Bar Time now", datetime=jdatetime.datetime(1380,8,2,12,12,12)).save()
-    >>> BarTime.objects.filter(datetime__lt= jdatetime.datetime(1380,8,2,12,12,12 ))
+    >>> BarTime.objects.filter(datetime__date=jdatetime.datetime(1380,8,2,12,12,12))
```

Comment:
Please add another example for `__date` filter with `jdatetime.date` and `str` value